packages/data-context/src/data/ProjectConfigIpc.ts

```diff
@@ -310,6 +319,17 @@ export class ProjectConfigIpc extends EventEmitter {
       debug(`no typescript found, just use regular Node.js`)
     }
 
+    if (process.env.CYPRESS_INTERNAL_E2E_TESTING_SELF_PARENT_PROJECT) {
```

Comment:
Bit of a shame we need to ship test code to prod :'(

This is "the change" that lets v8 snapshots run in cy-in-cy?

Reply:
Is there anyway to verify it now uses snapshots in the tests? How do we know it works (and catch it if stops?)

Reply:
Yeah it's unfortunate, but I'm not sure of a clean way to do it otherwise with how everything is structured.

And, I like your idea. I added a check to ensure that we fail in cypress in cypress if the snapshot is unavailable.